airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx

```diff
@@ -152,6 +153,7 @@ const WebHookForm: React.FC<WebHookFormProps> = ({ webhook, onSubmit, successMes
             </Cell>
             <Cell flex={1}>
               <Field name="sendOnSuccess">
+                {/* In this case, Headless UI onChange is not compatible with Formik field onChange */}
```

Comment:
Just put an note here that the `onChange` for formik is expecting the `event` object but our switch is sending just a boolean of the current state, so this most likely breaks.